<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlsDock</class>
 <widget class="QDockWidget" name="ControlsDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>L2 Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>Stop rotation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start rotation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnL2Disconnect">
       <property name="text">
        <string>L2 disconnect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnL2Connect">
       <property name="text">
        <string>L2 connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnReset">
       <property name="text">
        <string>L2 reset</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnVersion">
       <property name="text">
        <string>Version info</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnConfig">
       <property name="text">
        <string>Config</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
